lib/scripts/conversionScriptPart.js

```diff
@@ -76,6 +83,11 @@ page.onInitialized = function() {
 };
 
 page.open(body.url, function () {
+    
+    page.evaluate(function() {
+        console.log(JSON.stringify(document.cookie));
+    });
+
```

Comment:
I believe you could remove this code.
```js
page.evaluate(function() {
       console.log(JSON.stringify(document.cookie));
 });
```

 Instead pass it in test to avoid unnecessary logging in real run. See the following, it could work?

```js
it('should include user defined cookies in http requests', function(done) {
           conversion({
                html: '<script>console.log(JSON.stringify(document.cookie));</script>',
...
```